sdks/python/apache_beam/runners/interactive/pipeline_instrument.py

```diff
@@ -314,7 +307,6 @@ def cache_key(self, pcoll):
     if cacheable:
       return '_'.join((cacheable['var'],
                        cacheable['version'],
-                       cacheable['pcoll_id'],
```

Comment:
`pcoll_id` is not stable for cache_key especially with the coming introduction of `pipeline fragment` in `[BEAM-7926-part2]`. A combination of other fields is sufficient to locate a cache.

Reply:
Thanks! Perhaps put that explanation in the comment?

Reply:
Adding
```
    'pcoll_id' of cacheable is not stable for cache_key, thus not included in
    cache key. A combination of 'var', 'version' and 'producer_version' is
    sufficient to identify a cached PCollection.
```